app/vmctl: update importing tips when migrating data with overlapping time range (#3941)
VictoriaMetrics/VictoriaMetrics: app/vmctl/README.md

to the corresponding source address.
9. `vmctl` supports `--vm-native-disable-http-keep-alive` to allow `vmctl` to use non-persistent HTTP connections to avoid
error `use of closed network connection` when run a longer export.
10. Migrating data with overlapping time range for destination data can produce duplicates series at destination. 

> To avoid duplicates on the destination set `-dedup.minScrapeInterval=1ms`  for `vmselect` and `vmstorage`. 

Reviewer: ```suggestion
To avoid duplicates on the destination set `-dedup.minScrapeInterval=1ms` for `vmselect` and `vmstorage`. 
```

Author: fixed

> It will instruct `vmselect` and `vmstorage` to ignore duplicates with match exactly by the timestamp (ms is the highest timestamp precision in VictoriaMetrics)     

Reviewer: ```suggestion
This will instruct `vmselect` and `vmstorage` to ignore duplicates with matching timestamps.
```

Author: fixed
